src/lavinmq/clustering/client.cr

```diff
@@ -49,6 +53,12 @@ module LavinMQ
         if http_proxy = @http_proxy
           spawn http_proxy.forward_to(host, @config.http_port), name: "HTTP proxy"
         end
+        if unix_amqp_proxy = @unix_amqp_proxy
+          spawn unix_amqp_proxy.forward_to(host, @config.amqp_port, true), name: "AMQP proxy"
```

Comment:
This will set `@proxy_header = true` in `Proxy`, but `@server` is a `UNIXServer` so all clients will be `UNIXSocket`.

However, `@proxy_header == true` only affects `TCPSocket` clients in `#handle_client` so something is weird.

Or am I missing something?

Reply:
ah, yes, copy paste error. makes no sense to forward the local/remote address of the UNIX socket to the leader.